doc/manual/parallel-computing.rst

```diff
@@ -997,6 +997,79 @@ Keyword argument ``topology`` to ``addprocs`` is used to specify how the workers
 Currently sending a message between unconnected workers results in an error. This behaviour, as also the
 functionality and interface should be considered experimental in nature and may change in future releases.
 
+Multi-threading (Experimental)
+-------------------------------
+In addition to tasks, remote calls and remote references, Julia from v0.5 will natively support multi-threading. 
+
+### Setup 
+First, load the threading module into your workspace::
+
+    using Base.Threads
+
+By default, Julia starts up with a single thread of execution. This can be verified by 
+using the command :obj:`nthreads()`::
+
+    julia> nthreads()
+    1
+
+The number of threads Julia starts up with is controlled by an environment variable 
+called `JULIA_NUM_THREADS`. Now, let's start up Julia with 4 threads.::
+    export JULIA_NUM_THREADS=4
```

Comment:
Yeah, it works on other shells AFAIK. Maybe I'll mention people should use `set` instead of `export` on C shells.